src/reporter/reporter.types.ts

```diff
@@ -0,0 +1,103 @@
+/*
+   Copyright 2022 Mitch Spano
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+	 https://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+ */
+
+import { PluginInputs } from "../common";
+import { ScannerViolation } from "../sfdxCli";
+import { Context } from "@actions/github/lib/context";
+
+export type GithubCheckRun = {
```

Comment:
Is this used in the `comments-reporter`? If not, it seems like it belongs in `annotations-reporter` - right?

Reply:
Understandable, but I think the representation of the comment and the annotation/check run, doesn't have anything to do with the individual reporters. Makes more sense to me for them to be imported from a types file instead of the class file 🤔 But I can shift them if you prefer